server/server.go

```diff
@@ -34,12 +40,16 @@ func (b *Backend) Operator(ctx context.Context, req *pbVersion.OperatorRequest)
 }
 
 func (b *Backend) Apply(_ context.Context, req *pbVersion.ApplyRequest) (*pbVersion.VersionResponse, error) {
+	if req.Product == pmmServerProduct {
+		return nil, status.Error(codes.Unimplemented, "not implemented for pmm-server")
+	}
+
 	err := transformRequest(req)
 	if err != nil {
 		return nil, err
 	}
 
-	vs, err := operatorProductData(req.Product, req.OperatorVersion)
+	vs, err := operatorProductData("operator", req.Product, req.OperatorVersion)
```

Comment:
why do we change it to `operator`?
We already check for pmm at the beginning and we could forget to change it in future

Reply:
I am not sure what you mean. The signature of func `operatorProductData` changed to include `productFamily`. We don't implement Apply for PMM - the check in the beginning - so we can pass `productFamily` "opearator" right away. @qJkee